### PR TITLE
fix: remove setting inconsistent value change from updateFrom

### PR DESCRIPTION
**Pull request by yadamskaya**
Closes: https://github.com/exadel-inc/ui-playground/issues/646

### CODE REVIEW

File: src/plugins/settings/base-setting/base-setting.ts [hunk 1]
```diff
@@ -130,8 +118,8 @@ export abstract class UIPSetting extends UIPPlugin {
    * Disable setting
    * By default is used when there are no setting's targets
    */
-  public set disabled(force: boolean) {
-    this.$$attr('disabled', force);
+  public setDisabled(force: boolean): void {
+    return;
```
Comment (ala-n) [inline src/plugins/settings/base-setting/base-setting.ts:122]: Is it problem for us to have no disable attribute on the element itself ?

**Review by ala-n — APPROVED** [review]

File: src/plugins/settings/base-setting/base-setting.ts (hunk 1 above)
Comment (yadamskaya) [reply src/plugins/settings/base-setting/base-setting.ts:122]: @ala-n actually no, lets keep it

**Review by ala-n — APPROVED** [review]

**Pull request merged**